File: clients/javascript/src/models/kvSetOut.ts
// this file is @generated

export interface KvSetOut {
    /** Whether the operation succeeded or was a noop due to pre-conditions. */
    success: boolean;
}

export const KvSetOutSerializer = {
    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _fromJsonObject(object: any): KvSetOut {
        return {
            success: object['success'],
        };
    },

    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _toJsonObject(self: KvSetOut): any {
        return {
            'success': self.success,
        };
    }
}

File: clients/rust/src/models/kv_set_out.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct KvSetOut {
    /// Whether the operation succeeded or was a noop due to pre-conditions.
    pub success: bool,
}

impl KvSetOut {
    pub fn new(success: bool) -> Self {
        Self { success }
    }
}
